src/App.tsx

```diff
@@ -21,6 +30,11 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            onChange={(event) => {
+              setPerPage(+event.target.value);
+              setCurrentPage(1);
+            }}
```

Comment:
Move to separate function  (transfer the function to a variable, and here only write the name of the function)